elements/lisk-transaction-pool/src/types.ts

```diff
@@ -31,6 +31,7 @@ export interface TransactionObject {
 
 export interface TransactionFunctions {
 	getBytes: () => Buffer;
+	toJSON(): unknown;
```

Comment:
Don't we know all transaction types which can be used here with `|`

Reply:
Transaction pool doesn't know about any particular transaction